.changeset/kind-penguins-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': patch
```

Comment:
It does look like there are some changes to the `OAuthAdapterOptions` that potentially are breaking given that it's exported? Are there things we should highlight in the changeset? In any case it should probably be `minor` given that the package is below 1.0 and has type changes.

Reply:
@jhaals Ah yes, I made those changes as I felt the list of options were getting a bit large and it could make sense to group the cookie related ones in their own object. Didn't realize this was potentially breaking 😅  If the changes make sense I can add a note to the changset and change it to `minor`?